rpc_jobs/periodic_cleanup.yml

```diff
@@ -9,7 +9,7 @@
             Hours. Instances older than this will be removed.
       - string:
           name: "PROTECTED_PREFIX"
-          default: "long-running-slave|influx-|WEBHOOK-PROXY|webhook-proxy|nodepool|repo-server-"
+          default: "long-running-slave|influx-|WEBHOOK-PROXY|webhook-proxy|nodepool|repo-server-|ArtifactBuilder"
```

Comment:
I thought the hostnames needed to be lowercase otherwise it causes issues with some tool or another. Is this not affected in the same way as webhook-proxy?

Reply:
This is just a jenkins name, not a host name. The actual host name is quite different and is lower case.